modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/SemanticRequestMetrics.java

```diff
@@ -92,9 +101,15 @@ public void response(Response<ByteString> response) {
     timerContext.ifPresent(Timer.Context::stop);
 
     StatusType.Family family = response.status().family();
-    if (family == SERVER_ERROR) {
+    if (family != INFORMATIONAL && family != SUCCESSFUL) {
```

Comment:
redirects should not count as errors either

Reply:
The reason why I added two new metrics was that I didn't want to touch the error-ratio one (as was discussed in this PR https://github.com/spotify/apollo/pull/205). I'd prefer not to touch that metric.